Crane/app/src/main/java/androidx/compose/samples/crane/base/ExploreSection.kt

```diff
@@ -98,15 +89,15 @@ fun ExploreSection(
                             }
                         }
                     }
-                    item(span = {
+                    /*item(span = {
```

Comment:
What's this? If you're commenting this out, shouldn't you just remove it?

Reply:
Forgot about it - updated now to include it again.